jerry-core/ecma/operations/ecma-proxy-object.c

```diff
@@ -360,7 +360,7 @@ ecma_proxy_object_get_prototype_of (ecma_object_t *obj_p) /**< proxy object */
  * The Proxy object [[SetPrototypeOf]] internal routine
  *
  * See also:
- *          ECMAScript v6, 9.5.2
+ *          ES11+: 9.5.2
```

Comment:
Could we use the ES11 (or ECMAString v11) name?  Adding the "+" could mean anything and the newer standards could change. Having a fix version reference would be better.